src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,4 +11,11 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+
+    /* Policy Prefix definitions */
+    public static final Prefix PREFIX_POLICY_NAME = new Prefix("pn/");
+    public static final Prefix PREFIX_POLICY_START_DATE = new Prefix("d/");
+    public static final Prefix PREFIX_POLICY_PREMIUM = new Prefix("pr/");
+    public static final Prefix PREFIX_POLICY_FREQUENCY = new Prefix("fr/");
```

Comment:
I was thinking we could keep to the convention where we prefix all policy commands with a "p", i.e.

Start Date -> "pd/"
Frequency -> "pfr/" or "pf/"

Reply:
Okay! Good suggestion! I will create a new PR to make the changes, and update the User Guide!